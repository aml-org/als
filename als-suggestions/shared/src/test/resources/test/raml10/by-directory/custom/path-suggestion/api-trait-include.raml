#%RAML 1.0
traits:
  trait: !include /*

/.:
    get:
        is: [ trait ]