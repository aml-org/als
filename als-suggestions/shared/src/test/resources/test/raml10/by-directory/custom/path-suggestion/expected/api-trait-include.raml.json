[
  {
    "label": "/api-trait-inlined.raml",
    "kind": [
      
    ],
    "detail": "unknown",
    "documentation": "Path suggestion",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "11/api-trait-inlined.raml",
    "filterText": "/api-trait-inlined.raml",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 2,
          "character": 18
        },
        "end": {
          "line": 2,
          "character": 19
        }
      },
      "newText": "/api-trait-inlined.raml"
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "/api.raml",
    "kind": [
      
    ],
    "detail": "unknown",
    "documentation": "Path suggestion",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "11/api.raml",
    "filterText": "/api.raml",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 2,
          "character": 18
        },
        "end": {
          "line": 2,
          "character": 19
        }
      },
      "newText": "/api.raml"
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "/double-slash.raml",
    "kind": [
      
    ],
    "detail": "unknown",
    "documentation": "Path suggestion",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "11/double-slash.raml",
    "filterText": "/double-slash.raml",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 2,
          "character": 18
        },
        "end": {
          "line": 2,
          "character": 19
        }
      },
      "newText": "/double-slash.raml"
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "/expected/",
    "kind": [
      
    ],
    "detail": "unknown",
    "documentation": "Path suggestion",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "11/expected/",
    "filterText": "/expected/",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 2,
          "character": 18
        },
        "end": {
          "line": 2,
          "character": 19
        }
      },
      "newText": "/expected/"
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  }
]